Assets/MixedRealityToolkit/_Core/Inspectors/ControllerPopupWindow.cs

```diff
@@ -0,0 +1,755 @@
+        private static readonly GUIContent YAxisContent = new GUIContent("Y Axis", "Vertical Axis to listen for.");
+        private static readonly GUIContent InvertContent = new GUIContent("Invert", "Should an Axis be inverted?");
+        private static readonly GUIContent[] InvertAxisContent =
+        {
+            new GUIContent("None"),
+            new GUIContent("X"),
+            new GUIContent("Y"),
+            new GUIContent("Both")
+        };
+
+        private static readonly int[] InvertAxisValues = { 0, 1, 2, 3 };
+
+        private static readonly Vector2 InputActionLabelPosition = new Vector2(256f, 0f);
+        private static readonly Vector2 InputActionDropdownPosition = new Vector2(88f, 0f);
+        private static readonly Vector2 InputActionFlipTogglePosition = new Vector2(-24f, 0f);
+        private static readonly Vector2 HorizontalSpace = new Vector2(8f, 0f);
+
+        private static readonly Rect ControllerRectPosition = new Rect(new Vector2(128f, 0f), new Vector2(512f, 512f));
+
+        private static ControllerPopupWindow window;
+        private static ControllerInputActionOptions controllerInputActionOptions;
+
+        private static GUIContent[] axisLabels;
+        private static int[] actionIds;
+        private static GUIContent[] actionLabels;
+        private static int[] rawActionIds;
+        private static GUIContent[] rawActionLabels;
+        private static int[] digitalActionIds;
+        private static GUIContent[] digitalActionLabels;
+        private static int[] singleAxisActionIds;
+        private static GUIContent[] singleAxisActionLabels;
+        private static int[] dualAxisActionIds;
+        private static GUIContent[] dualAxisActionLabels;
+        private static int[] threeDofPositionActionIds;
+        private static GUIContent[] threeDofPositionActionLabels;
+        private static int[] threeDofRotationActionIds;
+        private static GUIContent[] threeDofRotationActionLabels;
+        private static int[] sixDofActionIds;
+        private static GUIContent[] sixDofActionLabels;
+        private static bool[] isMouseInRects;
+
+        private static bool editInputActionPositions;
+
+        private static float defaultLabelWidth;
+        private static float defaultFieldWidth;
+
+        private static Vector2 horizontalScrollPosition;
+
+        private SerializedProperty currentInteractionList;
+
+        private ControllerPopupWindow thisWindow;
+
+        private Handedness currentHandedness;
+        private SupportedControllerType currentControllerType;
+
+        private Vector2 mouseDragOffset;
+        private GUIStyle flippedLabelStyle;
+        private Texture2D currentControllerTexture;
+        private ControllerInputActionOption currentControllerOption;
+
+        private bool IsCustomController => currentControllerType == SupportedControllerType.GenericOpenVR ||
```

Comment:
this likely needs to change while we await guidance on controller images

Reply:
As above